lms/djangoapps/instructor/views/api.py

```diff
@@ -1038,31 +1038,39 @@ def get_problem_responses(request, course_id):
         "task_id": "4e49522f-31d9-431a-9cff-dd2a2bf4c85a"
     }
     ```
```

Comment:
Could you update this documentation to also point to the new API endpoints?

Reply:
@xitij2000 

Is [this](https://github.com/edx/edx-platform/pull/24873/files#diff-9030096cb9659025eaf82ed406dcb128R1008-R1011) what you meant?